tests/LimitOrderTest.php

```diff
@@ -46,9 +46,8 @@ public function testReverse()
         $ii->addExpression('total_gross', '[total_net]+[total_vat]');
         $ii->getField($ii->id_field)->system = false;
 
-        // pass parameters as CSV string
         $i = clone $ii;
-        $i->setOrder('total_net desc, total_gross desc');
```

Comment:
In few old projects there are a lot of such syntax, but ... well, I guess it have to go...

Reply:
This should be never allowed